emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "edges" : ecwb.WrapperBase.Access.WORM, # set of CommonTrips that include this location
+             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
```

Comment:
Yeah that's right

On Thursday, February 4, 2016, shankari <notifications@github.com> wrote:

> In emission/core/wrapper/common_place.py
> <https://github.com/e-mission/e-mission-server/pull/227#discussion_r51844466>
> :
>
> > @@ -0,0 +1,17 @@
> > +import logging
> > +import emission.core.wrapper.wrapperbase as ecwb
> > +
> > +class CommonPlace(ecwb.WrapperBase):
> > +    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
> > +             "edges" : ecwb.WrapperBase.Access.WORM, # set of CommonTrips that include this location
> > +             "successors" : ecwb.WrapperBase.Access.WORM, # set of CommonPlaces connected to this location
>
> Just so that I understand, we don't technically need the successors,
> right? We can just iterate through the edges and find the destination node
> for each of them. So this is a convenience function that is duplicated.
> Correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/227/files#r51844466>.
>